draft-birkholz-scitt-software-use-cases.md

```diff
@@ -167,6 +167,35 @@ There is no standardized way to:
 * to check for the trustworthiness of a provider
 * to check the integrity of modifications or transformations done by a provider
 
+## Verifying Firmware Provenance for large and complex systems
+
+In contrast to operating systems or user space software components of a large and complex systems, firmware components are often already executed during boot-cycles before there is an opportunity to authenticate them.
+
+Authentication takes place, for example, by checking an accompanying signature or by creating a hash of measurements and comparing them to a reference value.
+Corresponding procedures are often called authenticated, measured, or secure boot. The output of these high assurance boot procedures is often used as input to more complex remote attestation procedures.
+
+If measurements before execution are not possible, static after-the-fact analysis is required.
+
+In these cases measurements (e.g., a hash or digests) are stored in a protected or shielded environment (e.g., TEEs or TPMs) on these systems itself.
```

Comment:
```suggestion
In such cases measurements (e.g., a hash or digests) should be stored in a protected or shielded environment (e.g., TEEs or TPMs).
```

Reply:
agree on intention, but not sure its a fact... alternatives: 

"ought to be"

"when best practices are followed, are..."

"might..."

Reply:
Of the options above, I like "when best practices are followed, are..." best.

Reply:
From the option above, I like "when best practices are followed, are..." best.

Reply:
picked one suggestion as per https://github.com/ietf-scitt/draft-birkholz-scitt-software-supply-chain-use-cases/pull/17#issuecomment-1430206115